Resources/README.md

```diff
@@ -10,9 +10,6 @@
 * ### [WordPress Online Portfolio Tutorial](https://github.com/CIS320-team-3/CIS320-Team-3/blob/main/Resources/Tutorials/WordPress_for_Beginners.md)
   * This goes over the basics of WordPress and how to make a free website using it. With a focus on how to create portfolios using it, this is a valuable resource to check and use.
 
-* ### [How to Create and Host a Webpage on Github with Github Pages](https://github.com/CIS320-team-3/CIS320-Team-3/blob/main/Resources/Tutorials/Create_and_Host_Webpage_on_Github.md)
-  * [For those unfamiliar with Github or Github Pages, this is a step-by-step tutorial on how to create and host your webpage on the site.](https://youtu.be/2MsN8gpT6jY)
```

Comment:
This should not be entirely deleted.  If you can remove the URL reference on the header `How to Create and Host a Webpage on Github with Github Pages`, everything else should be good as is!